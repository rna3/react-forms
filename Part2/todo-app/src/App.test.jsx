import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test("renders App without crashing", () => {
  render(<App />);
});

test("matches snapshot for App", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });