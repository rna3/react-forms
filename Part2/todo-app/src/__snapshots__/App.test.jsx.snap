// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for App 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Todo App
    </h1>
    <div>
      <form>
        <input
          placeholder="New task"
          type="text"
          value=""
        />
        <button>
          Add
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
